### PR TITLE
Feat/users list from odoo and UI

### PR DESCRIPTION
**Pull request by stecb**
Handled a new state to represent the users list from odoo, and designed/coded the UI to show the users in the resolution list/details/form.

### CODE REVIEW

File: src/components/ResolutionUser.svelte [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+          >
+        </Wrapper>
+      {/if}
+    {/if}
+    {#if $signerAddress?.toLowerCase() === ethereumAddress.toLowerCase() && !hideInfo}
+      <Tag label="you" size={size === "sm" ? "xs" : "sm"} />
+    {/if}
+  </span>
+{:else}
+  <div class={`wrapper ${hasBg ? "wrapper--has-bg" : ""} wrapper--${size}`}>
```
Comment (vrde) [inline src/components/ResolutionUser.svelte:53]: With svelte you can do `class:wrapper--has-bh={hasBg}` (see [doc](https://svelte.dev/docs#template-syntax-element-directives-class-name)).

File: src/components/Tag.svelte [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <script lang="ts">
   export let label: string;
-  export let size: "sm" | "md" | "lg" = "md";
+  export let size: "xs" | "sm" | "md" | "lg" = "md";
 </script>
 
-<span class={`tag tag--${label} tag--${size}`}>
+<span class={`tag tag--${label.toLowerCase()} tag--${size}`}>
```
Comment (vrde) [inline src/components/Tag.svelte:6]: This should work as well, and it's slightly more readable imo: `class="tag tag--{label.toLowerCase()} tag--{size}"`

**Review by vrde — APPROVED** [review]
Looks great! I've added two minor comments, up to you to decide what to do.

File: src/components/ResolutionUser.svelte (hunk 1 above)
Comment (stecb) [reply src/components/ResolutionUser.svelte:53]: Ohh nice. Now that I'm thinking I was also used to use https://github.com/JedWatson/classnames (w/ react I've mainly used styled components lately)

File: src/components/Tag.svelte (hunk 2 above)
Comment (stecb) [reply src/components/Tag.svelte:6]: Of course, thanks

**Pull request merged**